test/functional/test_framework/blocktools.py

```diff
@@ -48,6 +48,7 @@
 
 MAX_BLOCK_SIGOPS = 20000
 MAX_BLOCK_SIGOPS_WEIGHT = MAX_BLOCK_SIGOPS * WITNESS_SCALE_FACTOR
+MAX_STANDARD_TX_WEIGHT = 400000
```

Comment:
nit: could replace [this line](https://github.com/bitcoin/bitcoin/blob/2d21060af831fa8f8f1791b4464961d5f28a88cb/test/functional/p2p_segwit.py#L1265) with this constant.

Reply:
leaving this as is, as I suspect there might be other places that needs to be replaced also.
this and others can come in a separate PR ?